### PR TITLE
update brief

### PR DESCRIPTION
**Pull request by melissajschaff**

### CODE REVIEW

File: products/health-care/appointments/va-online-scheduling/express-care-research/express-care-project-brief.md [hunk 1]
```diff
@@ -1,76 +1,70 @@
...
+As a Veteran who lives in a geographical COVID-19 hot spot, I need to be provided a way get immediate COVID-19 screening and evaluation from my VA doctor/PACT team.
+
+As a VA facility located in a COVID-19 hot spot, I need to be able to screen and evaluate VA patients through a digital medium so that I can limit exposure of VA staff and patients to potential COVID-19 cases and/or alleviate need for in-person evaluation this is creating operational bottlenecks. I want to do this by PACT.
 
-Currently, "Express Care" is a care option developed by VA for immediate, same day evaluation via VA Video Connect (VA's proprietary telehealth platform). However, Express Care is not utilized by VA providers or patients and we believe the business rules in place prevent this platform from being a preferred option by providers and patients, alike.
+### Problem statements, additional [maybe remove, because not talking to VA or VA providers]
```
Comment (lenaecb) [inline products/health-care/appointments/va-online-scheduling/express-care-research/express-care-project-brief.md:25]: I think we keep this, but I think we can expand on later. I believe focus for now should remain on the Veterans.

**Review by lenaecb — APPROVED** [review]
Just some tiny points of feedback, but nothing big.

File: products/health-care/appointments/va-online-scheduling/express-care-research/express-care-project-brief.md [hunk 2]
```diff
@@ -1,76 +1,70 @@
...
+- How do Veterans seek same-day / urgent care virtually? Can they do this currently through the VA?
+- What type of devices/platforms do Veterans use for video chats (if at all)?
+- Are Veterans comfortable getting medical advice via Telehealth? If so, what for? If not, why?
+- What VA health care resources are Veterans aware of related to COVID-19?
+- Will Veterans use a VA online COVID-19 screening tool?
+- Will Veterans choose Express Care for COVID-19 screening?
+- When do Veterans think COVID-19 Express Care appointments be available to them?
 
+## Timeline:
+- ***Friday, 4/3 have researched/tested options for rapid implementation within VAOS on directing Veterans to immediate screening and/or care due to COVID-19***
```
Comment (lenaecb) [inline products/health-care/appointments/va-online-scheduling/express-care-research/express-care-project-brief.md:67]: I would clarify here that it's being tested with 'legacy' VAOS, not VAOS-R (new VAOS on VA.gov)

**Review by lenaecb — APPROVED** [review]

File: products/health-care/appointments/va-online-scheduling/express-care-research/express-care-project-brief.md (hunk 1 above)
Comment (melissajschaff) [reply products/health-care/appointments/va-online-scheduling/express-care-research/express-care-project-brief.md:25]: I'm going to move the provider problem statements to the bottom and call it "out of scope problem statements" or something like that

File: products/health-care/appointments/va-online-scheduling/express-care-research/express-care-project-brief.md [hunk 3]
```diff
@@ -1,76 +1,70 @@
...
+# Express Care Project Brief
 
-## Problem Statement:
-COVID-19 is creating hot spots throughout the country. These hot spots put a strain on VA medical facilities and staff as they try and field calls and screen patients.
+## Overview
+Veterans have medical needs that may either be resolved without an in-person appointment and/or want guidance about how seek additional care for a medical concern, yet may not warrant a trip to the ER. For various reasons, the Veteran may not be able to go into a physical location for this care. In both cases, however, the Veteran needs to talk to a medical professional,  today.
 
-As a Veteran who lives in a geographical COVID-19 hot spot, I need to be provided a way get immediate COVID-19 screening and evaluation from my VA doctor/PACT team.
+"Express Care" is a solution developed by VA (launched in 2017) for immediate, same day evaluation via VA Video Connect (VA's proprietary Telehealth platform). However, Express Care is not utilized by VA providers, and as a result cannot be used by Veteran patients.
```
[the remark this answers is not in the document]
Comment (melissajschaff) [reply products/health-care/appointments/va-online-scheduling/express-care-research/express-care-project-brief.md:6]: right! its phone call _before_ video visit!
 
+ phone call triage by a scheduler...

File: products/health-care/appointments/va-online-scheduling/express-care-research/express-care-project-brief.md (hunk 2 above)
Comment (melissajschaff) [reply products/health-care/appointments/va-online-scheduling/express-care-research/express-care-project-brief.md:67]: yes, and that the timeline is undetermined ATM.

File: products/health-care/appointments/va-online-scheduling/express-care-research/express-care-project-brief.md [hunk 4]
```diff
@@ -1,76 +1,70 @@
-# Iteration on Research - Focus COVID-19 Response
-Based on urgency to provide additional COVID-19 response coverage through digital channels, we need to re-focus Express care research to specifically solve for directing verteran to appropriate digital user flow for COVID-19 screening and care. 
+# Express Care Project Brief
 
-## Problem Statement:
-COVID-19 is creating hot spots throughout the country. These hot spots put a strain on VA medical facilities and staff as they try and field calls and screen patients.
+## Overview
+Veterans have medical needs that may either be resolved without an in-person appointment and/or want guidance about how seek additional care for a medical concern, yet may not warrant a trip to the ER. For various reasons, the Veteran may not be able to go into a physical location for this care. In both cases, however, the Veteran needs to talk to a medical professional,  today.
```
[the remark this answers is not in the document]
Comment (melissajschaff) [reply products/health-care/appointments/va-online-scheduling/express-care-research/express-care-project-brief.md:4]: resolved! closing this comment

File: products/health-care/appointments/va-online-scheduling/express-care-research/express-care-project-brief.md [hunk 5]
```diff
@@ -1,76 +1,71 @@
...
+As a Veteran who lives in a geographical COVID-19 hot spot, I need to be provided a way get immediate COVID-19 screening and evaluation from my VA doctor/PACT team. 
 
-Currently, "Express Care" is a care option developed by VA for immediate, same day evaluation via VA Video Connect (VA's proprietary telehealth platform). However, Express Care is not utilized by VA providers or patients and we believe the business rules in place prevent this platform from being a preferred option by providers and patients, alike.
+As a VA facility located in a COVID-19 hot spot, I need to be able to screen and evaluate VA patients through a digital medium so that I can limit exposure of VA staff and patients to potential COVID-19 cases and/or alleviate need for in-person evaluation this is creating operational bottlenecks. I want to do this by PACT.
```
Comment (lalexanderson-dsva) [inline products/health-care/appointments/va-online-scheduling/express-care-research/express-care-project-brief.md:23]: I think the nurse calling them back through this model may not necessarily be part of their PACT. ANy follow-up appointments would be.

File: products/health-care/appointments/va-online-scheduling/express-care-research/express-care-project-brief.md [hunk 6]
```diff
@@ -1,76 +1,71 @@
...
 
-Additionally, with the recent outbreak of the COVID-19 pandemic, there is an urgent need to give Veterans fast, safe access to care and evaluation via telehealth either as an alternative to in office, face-to-face visits. The VAOS team believes Express Care can serve as a tool to facilitate immediate access to remote medical care as well as provide VA providers a remote mechanism for screening patients who suspect they have COVID-19.
+As a VA medical provider, the business rules that dictate scheduling for Express Care prevent me from using the platform as a way to treat my patients. 
 
 ## Goal:
 We want to perform usability research with Veterans to learn more about:
- + Veterans undertanding of what appointment types are available to them through VA, specifically for telemedicine and same-day appointments
-  + how might Veterans seek same-day care (either through VA or other sources)
-  + how might Veterans go about receiving immediate care at the VA, with emphasis on understanding use of 'telehealth' or online care options. 
+- Veterans understanding of what appointment types are available to them through VA, specifically for telemedicine and same-day appointments
```
Comment (lalexanderson-dsva) [inline products/health-care/appointments/va-online-scheduling/express-care-research/express-care-project-brief.md:33]: I think the phrasing "types" will be confusing to Veterans. type of care = specialty; "different WAYS (modalities) to receive care" = in person/phone/video, etc.

**Pull request closed without merging**